project/AppDependencies.scala

```diff
@@ -1,40 +1,42 @@
-import play.core.PlayVersion
+import play.core.PlayVersion.current
 import sbt._
 
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"           %% "bootstrap-play-25"  % "4.9.0",
-    "uk.gov.hmrc"           %% "play-ui"            % "7.32.0-play-25",
-    "uk.gov.hmrc"           %% "metrix"             % "3.11.0-play-25",
-    "uk.gov.hmrc"           %% "work-item-repo"     % "6.10.0-play-25",
+    "uk.gov.hmrc"           %% "bootstrap-backend-play-27" % "2.23.0",
+    "uk.gov.hmrc"           %% "play-ui"                    % "8.12.0-play-27",
+    "uk.gov.hmrc"           %% "metrix"                     % "4.7.0-play-27",
+    "uk.gov.hmrc"           %% "work-item-repo"             % "7.10.0-play-27",
     /*
      * metrix & work-item-repo depend on simple-reactivemongo but the play-25 branches are no longer maintained.
      * We add an explicit dependency below to force a version uplift.
      * This can be removed upon uplifting this project to Play 2.7
```

Comment:
Can we do this now?

Reply:
personally i'd rather keep the explicit version of simple-reactive mongo since its easier to tell what version its actually using. ill remove the comment

Reply:
Although it doesn't tell you what version metrix & work-item-repo require - they may have a later version which makes the simple-reactive version ineffective, or they may require an incompatible version..